src/examples/deploy_with_setup.yml

```diff
@@ -0,0 +1,14 @@
+description: |
+  A workflow to deploy your project using waypoint, with setup steps.
+usage:
+  version: 2.1
+  orbs:
+    waypoint: circleci/waypoint@x.y # This version number refers to the version of the orb, not the version of NodeJS
```

Comment:
Is the node comment still relevant?

Reply:
Good catch, keep meaning to remove this.